interactions.js

```diff
@@ -1,53 +1,10 @@
-var hover = (function()
-{
-    function mouseMoveF(e)
-    {
-        if(!this.show && this.classList.contains('hidden'))
-        {
-            this.classList.remove('hidden');
-            return;
-        }
+$('document').ready(function(){
 
-        if(this.show.classList.contains('hidden'))
-        {
-            this.show.classList.remove('hidden');
-            return;
-        }
-    }
+  $('.toggle_nav').on('mouseover', function(){
+    $(this).children(':nth-child(2)').removeClass('hidden');
+  });
+  $('.toggle_nav').on('mouseleave', function(){
+    $(this).children(':nth-child(2)').addClass('hidden');
+  });
 
-    function mouseOutF(e)
```

Comment:
Good work. This could be reduced *even* further with the `hover` event and `toggleClass` method.